src/targets/Jumper_2400.ini

```diff
@@ -14,6 +14,17 @@ extends = env:Jumper_AION_2400_T-Pro_TX_via_UART
 [env:Jumper_AION_2400_T-Pro_TX_via_ETX]
 extends = env:Jumper_AION_2400_T-Pro_TX_via_UART
 
+[env:Jumper_AION_2400_T-Lite_TX_via_UART]
+extends = env:Unified_ESP32_2400_TX_via_UART
+board = pico32
+board_config = jumper.tx_2400.t-lite
+
+[env:Jumper_AION_2400_T-Lite_TX_via_WIFI]
+extends = env:Jumper_AION_2400_T-Lite_TX_via_UART
+
+[env:Jumper_AION_2400_T-Lite_TX_via_ETX]
+extends = env:Jumper_AION_2400_T-Lite_TX_via_UART
+
```

Comment:
These are no longer required we you can select the Unified TX target and then choose the layout to attach to the firmware at the end of the build process.

Reply:
Do we expect users to manually add required config files?

Reply:
No, not at all. Hopefully, when we release Configurator will be fully supporting selecting targets from the targets.json file. Then these PIO based targets become pretty useless.
If configurator will not be ready then we can add these targets to PIO. The problem is that there are so many now and the number is growing at an alarming rate!